### ISSUE TITLE
jsBase64 undefined with v2.1.4

### ISSUE THREAD

**Issue by drkgreyhawk** [heroiclabs/nakama-js#88]
Upon updating to v2.1.4 we were getting an error when trying to authenticate with Email. Installed by downloading the file from release and importing it as specified through the [docs](https://heroiclabs.com/docs/javascript-client-guide/). We were able to authorize with 2.1.3. Calling jsBase64 from console returns `undefined`.

Error:
```
An error occured while authorizing: TypeError: Cannot read property 'encode' of undefined
    at NakamaApi.doFetch (nakama-js.umd.js:567)
    at NakamaApi.authenticateEmail (nakama-js.umd.js:677)
    at Client.authenticateEmail (nakama-js.umd.js:2062)
    at registerUser (nakama-interface.js:7)
    at Array.<anonymous> (main.js:18)
    at Function.f (elm.compiled.js:2219)
    at A3 (elm.compiled.js:59)
    at Object.b (elm.compiled.js:1978)
    at _Scheduler_step (elm.compiled.js:1822)
    at _Scheduler_enqueue (elm.compiled.js:1784)
```

Local Scope with breakpoint at line nakama-js.umd.js:567:
```
Local
    body: "{"email":"mail@example.com","password":"password1234"}"
    descriptor:
        configurable: true
        enumerable: true
        get: ƒ withCredentials()
        set: ƒ withCredentials()
        __proto__: Object
    fetchOptions: {method: "POST", headers: {…}}
    method: "POST"
    options:
        __proto__: Object
    queryParams:
        create: false
        username: "none"
        __proto__: Object
    this: NakamaApi
        configuration:
            basePath: "http://127.0.0.1:7350"
            password: ""
            timeoutMs: 7000
            username: "defaultKey"
            __proto__: Object
        __proto__: Object
    urlPath: "/v2/account/authenticate/email"
    urlQuery: "?create=false&username=none&"
    _this: NakamaApi
        configuration:
            basePath: "http://127.0.0.1:7350"
            password: ""
            timeoutMs: 7000
            username: "defaultKey"
            __proto__: Object
        __proto__: Object
```

**Comment by lugehorsam** [comment]
Thanks @drkgreyhawk . This is an issue specific to the `.umd.js` distribution. In the meantime if you import our package from npm, that should do the trick: https://www.npmjs.com/package/@heroiclabs/nakama-js

**Comment by lugehorsam** [comment]
This was resolved in the latest release, v2.1.5
